src/main/java/easytests/api/v1/controllers/IssuesController.java

```diff
@@ -69,4 +70,30 @@
      * delete(issueId)
      */
 
+    @DeleteMapping("/{issueId}")
+    public void delete(@PathVariable Integer issueId) throws NotFoundException, ForbiddenException {
+        final IssueModelInterface issueModel = this.getIssueModel(issueId);
+
+        if (issueModel == null) {
+            throw new NotFoundException();
+        }
+        if (!this.acl.hasAccess(issueModel)) {
+            throw new ForbiddenException();
+        }
+        this.issuesService.delete(issueModel);
+    }
+
+    private IssueModelInterface getIssueModel(Integer id) throws NotFoundException {
```

Comment:
Lets use getIssueModel for retrieve model for read and update.
For delete (cause it is the only one method) we could use direst option object set in delete action.